### PR TITLE
Add Application Insights to the ASP.NET 5 MVC version of MusicStore

### PR DESCRIPTION
**Pull request by olegsych**
@SergKanz, @abaranch, @upendras, @Sergei-Nikitin
I created this private fork of the aspnet/MusicStore because I don't have permissions to create a fork in the microsoft organization (I'm not sure we'd want one even if we could). Please use this pull request to provide feedback on the instrumentation of the MusicStore app. We can submit a pull request back to aspnet once we feel its ready.

Open Questions
=============
* Do we want to instrument the single-page (.spa) flavor of the MusicStore project? I'm not sure it actually works.
* Do we want to instrument the MVC flavors of the MusicStore projects?

### CODE REVIEW

File: NuGet.Config [hunk 1]
```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <add key="NuGet" value="https://nuget.org/api/v2/" />
     <add key="AspNetMaster" value="https://www.myget.org/F/aspnetmaster/api/v2" />
+    <add key="Application Insights" value="http://appinsights-aspnet.azurewebsites.net/nuget/" />
```
Comment (olegsych) [inline NuGet.Config]: Do we want to change this to a myget feed?

File: src/MusicStore/config.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 ﻿{
+    "ApplicationInsights": {
+        "InstrumentationKey": "2e7ea29b-5f84-4671-9648-677313cd796a"
```
Comment (olegsych) [inline src/MusicStore/config.json]: This is a key for an AI resource in my personal subscription I used to verify that instrumentation is working. Do we want to leave an actual key here? Whose subscription would we use?

File: NuGet.Config (hunk 1 above)
Comment (SergKanz) [inline NuGet.Config]: @Sergei-Nikitin, can you please investigate moving to MyGet as part of CI research

File: src/MusicStore/config.json (hunk 2 above)
Comment (SergKanz) [inline src/MusicStore/config.json]: I think you inject dummy key and for demo you'll need to enter an actual key from your account. We don't have spare account to share

Comment (olegsych) [inline src/MusicStore/config.json]: Do you mean something like YOUR_KEY or an actual GUID, like the one used now?

File: NuGet.Config (hunk 1 above)
Comment (olegsych) [inline NuGet.Config]: We can create a free/public myget feed and start using it now, and eventually have CI build pushing packages to it.

**Comment by olegsych** [comment]
Abandoning. Starting a new pull request based on the RC version of ASP.NET 5.

**Pull request closed without merging**